spring-cloud-stream-app-descriptor/src/main/resources/1.0.0.BUILD-SNAPSHOT/kafka-apps-docker.properties

```diff
@@ -0,0 +1,46 @@
+source.syslog=docker://org.springframework.cloud.stream.app:syslog-source-kafka:1.0.0.M1
+source.tcp=docker://org.springframework.cloud.stream.app:tcp-source-kafka:@project.version@
+source.time=docker://org.springframework.cloud.stream.app:time-source-kafka:@project.version@
+source.trigger=docker://org.springframework.cloud.stream.app:trigger-source-kafka:1.0.0.M1
+source.triggertask=docker://org.springframework.cloud.stream.app:triggertask-source-kafka:@project.version@
+source.twitterstream=docker://org.springframework.cloud.stream.app:twitterstream-source-kafka:@project.version@
+processor.bridge=docker://org.springframework.cloud.stream.app:bridge-processor-kafka:@project.version@
+processor.filter=docker://org.springframework.cloud.stream.app:filter-processor-kafka:@project.version@
+processor.groovy-filter=docker://org.springframework.cloud.stream.app:groovy-filter-processor-kafka:@project.version@
+processor.groovy-transform=docker://org.springframework.cloud.stream.app:groovy-transform-processor-kafka:@project.version@
+processor.httpclient=docker://org.springframework.cloud.stream.app:httpclient-processor-kafka:@project.version@
+processor.pmml=docker://org.springframework.cloud.stream.app:pmml-processor-kafka:@project.version@
+processor.scriptable-transform=docker://org.springframework.cloud.stream.app:scriptable-transform-processor-kafka:@project.version@
+processor.splitter=docker://org.springframework.cloud.stream.app:splitter-processor-kafka:@project.version@
+processor.transform=docker://org.springframework.cloud.stream.app:transform-processor-kafka:@project.version@
+sink.aggregate-counter=docker://org.springframework.cloud.stream.app:aggregate-counter-sink-kafka:@project.version@
+sink.cassandra=docker://org.springframework.cloud.stream.app:cassandra-sink-kafka:1.0.0.M1
+sink.counter=docker://org.springframework.cloud.stream.app:counter-sink-kafka:@project.version@
+sink.field-value-counter=docker://org.springframework.cloud.stream.app:field-value-counter-sink-kafka:@project.version@
+sink.file=docker://org.springframework.cloud.stream.app:file-sink-kafka:@project.version@
+sink.ftp=docker://org.springframework.cloud.stream.app:ftp-sink-kafka:@project.version@
+sink.gemfire=docker://org.springframework.cloud.stream.app:gemfire-sink-kafka:1.0.0.M1
+sink.gpfdist=docker://org.springframework.cloud.stream.app:gpfdist-sink-kafka:1.0.0.M1
+sink.hdfs=docker://org.springframework.cloud.stream.app:hdfs-sink-kafka:@project.version@
+sink.hdfs-dataset=docker://org.springframework.cloud.stream.app:hdfs-dataset-sink-kafka:1.0.0.M1
+sink.jdbc=docker://org.springframework.cloud.stream.app:jdbc-sink-kafka:@project.version@
+sink.log=docker://org.springframework.cloud.stream.app:log-sink-kafka:@project.version@
+sink.rabbit=docker://org.springframework.cloud.stream.app:rabbit-sink-kafka:@project.version@
+sink.redis=docker://org.springframework.cloud.stream.app:redis-sink-kafka:1.0.0.M1
```

Comment:
Let me quote myself from Slack:
> 
Artem Bilan [3:05 PM]  
According to the `RedisSinkConfiguration.redisSinkMessageHandler()` logic it is really a `topic` by default
To make it as `RedisStoreWritingMessageHandler` you have to specify `key`
To right via queue manner that must have `queue` property
Otherwise, that's correct: with `RedisPublishingMessageHandler` we will see data only if there is some subscriber for the `topic` on other side

Where JavaDocs for the first one is like:

```
* Implementation of {@link MessageHandler} which writes Message data into a Redis store
* identified by a key {@link String}.
```

So, assumption that `redis sink` is just `messaging` is wrong.

M-m-m. I see @markfisher has changed his mind already :smile: